lib/ex_doc.ex

```diff
@@ -25,6 +25,7 @@ defmodule ExDoc do
       project: nil,
       retriever: ExDoc.Retriever,
       source_beam: nil,
+      source_ref: "master",
```

Comment:
shouldn't the struct reflect what can be passed via config, such as CLI or the `docs` task?
https://github.com/elixir-lang/ex_doc/blob/05eb89e93559e129744cf021d580f14cf6fff313/lib/mix/tasks/docs.ex#L62